src/api/GraphSchema/types.ts

```diff
@@ -54,7 +54,8 @@ export enum PropertyTypeName {
   DATE = 'date',
   DATETIME = 'datetime',
   NUMBER = 'number',
-  STRING = 'string'
+  STRING = 'string',
+  CURRENCY = 'currency'
```

Comment:
nit: conforming to the alphabetical ordering enhances readability, especially as the enum grows. We might consider moving the `CURRENCY` value between `BOOLEAN` and `DATE`

Reply:
Thanks, done :)